packages/v4/patternfly-docs/content/training/html-css/fundamentals.md

```diff
@@ -0,0 +1,182 @@
+---
+id: HTML fundamentals training
+section: training
+hideNavItem: true
+---
+import { Card, CardBody, ClipboardCopy, Divider, PageSection } from '@patternfly/react-core';
+import { CopyCodeBlock } from '../copyCodeBlock/copyCodeBlock';
+
+<PageSection variant="light">
+# Fundamentals
+
+PatternFly Core is based on the principles of <a href="http://bradfrost.com/blog/post/atomic-web-design/" target="_blank">Atomic design</a> and <a href="http://getbem.com/introduction/" target="_blank">BEM</a> (Block, Element, Modifier).
+
+Atomic design is a methodology for creating design systems. In Atomic Design user interfaces are broken down into reusable components that can be re-assembled to create reusable interaction patterns.
+PatternFly consists of isolated and modular structures that fall into three categories:
+- Components are modular and independent structures concerned with presentation
+- Layouts allow for organizing and grouping their immediate children on the page
+- Demos illustrate how to assemble complex structures with components and layouts only
+
+## Components
+In PatternFly, components which cannot be broken down into smaller parts are known as the basic building blocks of user interfaces. Examples include the button, label and badge components. A codesandbox link has been provided to experiment with and follow the proceeding steps.
+
+<Card style={{maxWidth: "400px"}} component="div" isSelectableRaised onClick={() => window.open("https://codesandbox.io/s/html-fundamentals-start-hy2erg?file=/index.html", '_blank')}>
+  <CardBody>Codesandbox - HTML fundamentals</CardBody>
+</Card>
+</PageSection>
+<PageSection>
+### Step 1. Add a button component.
+In the `<body>` of the index.html file, add a button element, and apply the PatternFly styles to it so that it is a PatternFly button by adding the class `pf-c-button`. Apply the ‘primary’ styles to the button to make it a bright blue color by adding the `pf-m-primary` class to the button as well.
```

Comment:
```suggestion
In the `<body>` of the `index.html` file, add a button element, and apply the PatternFly styles to it so that it is a PatternFly button by adding the class `pf-c-button`. Apply the ‘primary’ styles to the button to make it a bright blue color by adding the `pf-m-primary` class to the button as well.
```